docs/upgrade/index.rst

```diff
@@ -11,24 +11,30 @@ Upgrade ScyllaDB
    ScyllaDB Open Source to ScyllaDB Enterprise <upgrade-to-enterprise/index>
    ScyllaDB AMI <ami-upgrade>
 
+Overview
+---------
 
 ScyllaDB upgrade is a rolling procedure - it does not require a full cluster shutdown and is performed without any 
 downtime or disruption of service.
 
-To ensure a successful upgrade and avoid breaking anything, you should perform your upgrades consecutively - to each 
-successive X.Y version. For example, to upgrade from version 4.4 to 5.0, you should first upgrade ScyllaDB to version 
-4.5, next from 4.5 to 4.6, and finally, from 4.6 to 5.0, **without skipping any major version**.
+To ensure a successful upgrade, follow the :ref:`documented upgrade procedures <upgrade-upgrade-procedures>` tested by ScyllaDB. This means that:
 
-.. raw:: html
+* You should perform the upgrades consecutively - to each successive X.Y version, **without skipping any major or minor version**.
+* Before you upgrade to the next version, the whole cluster (each node) must be upgraded to the previous version.
 
+Example
+========
 
-   <div class="panel callout radius animated">
-            <div class="row">
-              <div class="medium-3 columns">
-                <h5 id="getting-started">Procedures for upgrading ScyllaDB</h5>
-              </div>
-              <div class="medium-9 columns">
+The following example shows the upgrade path for a 3-node cluster from version 4.3 to version 4.6: 
 
+#. Upgrade all three nodes to version 4.4.
+#. Upgrade all three nodes to version 4.5.
+#. Upgrade all three nodes to version 4.6.
```

Comment:
what about the patch version?
should one upgrade to the latest patch version for each minor version?

Reply:
My understanding was that upgrading to a patch version is NOT necessary; that's why I added the general statement  "without skipping any major or minor version".
The example shows major and minor versions, without the patch versions.

Reply:
But there are no "major.minor" releases.
We always have a patch version (we start with major.minor.0 and go from there).
I presume we recommend to always upgrade to the latest patch release.

Reply:
we call major.minor.0 a major.minor release.
Agree we should clerify the fact there is no need to upgrade to the latest *patch* release

Reply:
I think the recommendation should be to upgrade through the latest patch release in each minor version since it might contain fixes related to the upgrade process, that otherwise, in their absence, they can botch the rolling upgrade.

Reply:
I see your point, and I'll add the information about the patch releases. Please review it before I update the source file. But please consider the following:
- This PR is supposed to help the users who keep asking the question: "can I upgrade directly from an old release", for example, from 4.4 to 4.6? The answer is "no, you need to upgrade to 4.5 first". I'd like to avoid overcomplicating things because this documentation section will not be helpful.
- The major objective is to help the users choose the upgrade guide they should follow. The names of the guides don't mention the patch release, for example, "ScyllaDB 4.4 to 4.5", or "ScyllaDB 4.5 to 4.6".
- When you upgrade with APT (and I believe most of our users use Ubuntu or image), you automatically upgrade to the latest patch release. Recently, I was asked to add [a note about how NOT to upgrade to the latest patch release](https://github.com/scylladb/scylladb/pull/12660). The details are in [this issue](https://github.com/scylladb/scylla-enterprise/issues/2176) - it's about Enterprise, but it's the same story with OSS.
-----------------------------
Update (for your review):
Under the existing example, I'd like to add the following:

Upgrading to each patch version by following the [Scylla Maintenance Release Upgrade Guide](https://docs.scylladb.com/stable/upgrade/upgrade-opensource/upgrade-guide-from-4.x.y-to-4.x.z/index.html) is optional. However, we recommend upgrading to the latest patch release for your version before upgrading to a new version. For example, upgrade to patch 4.4.8 before upgrading to version 4.5.

Reply:
Thanks, I've pushed the commit.
@bhalevy Could you do the final review so that this PR can be merged? Thanks!